test/plugin/scenarios/elasticsearch-7.x-scenario/configuration.yml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+type: jvm
+entryService: http://localhost:8080/elasticsearch-case/case/elasticsearch
+healthCheck: http://localhost:8080/elasticsearch-case/case/healthCheck
+startScript: ./bin/startup.sh
+framework: elasticsearch-7.x-scenario
+environment:
+- elasticsearch.server=elasticsearch-server:9200
+dependencies:
+  elasticsearch-server:
+    image: elasticsearch:7.0.0
```

Comment:
1. should be renamed.
2. uses ${CASE_SERVER_IMAGE_VERSION} instead of 7.0.0, please